media/media_options.gni

```diff
@@ -57,11 +57,6 @@ declare_args() {
     media_use_libvpx = false
   }
 
-  # Chrobalt doesn't use ffmpeg.
```

Comment:
Can we instead leave a `media_use_ffmpeg = true` in or around https://source.chromium.org/chromium/chromium/src/+/main:media/BUILD.gn;l=187 ?

Reply:
It is true by default. See https://github.com/youtube/cobalt/blob/03ed8d2f890c3912a2993730a0e5da9b52f3af55/media/media_options.gni#L48

We will need to figure it out if we want to disable it without breaking `media_unittests`.